### PR TITLE
Mobile search

### PR DESCRIPTION
**Pull request by skliarzfort**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: css/components/_search-section.scss [hunk 1]
```diff
@@ -80,4 +79,52 @@
             }
         }
     }
+
+    &__fake {
+        display: none;
```
Comment (tolerants) [inline css/components/_search-section.scss:84]:
```suggestion
        display: none;
        
```

File: css/components/_search-section.scss [hunk 2]
```diff
@@ -80,4 +79,52 @@
             }
         }
     }
+
+    &__fake {
```
Comment (tolerants) [inline css/components/_search-section.scss:83]: What is `fake`?

File: js/algolia-suggestions.js [hunk 3]
```diff
@@ -56,3 +60,8 @@ const Suggestions = {
         });
     }
 };
+
+$(function(){
```
Comment (tolerants) [inline js/algolia-suggestions.js:64]: Why do you need it?

File: js/customscripts.js [hunk 4]
```diff
@@ -33,8 +33,69 @@ $( document ).ready(function() {
...
+        close = $('.js-search-popup-close'),
+        body = jQuery('body'),
+        input = $('.search-input.aa-input');
+
+    // mobile-overflow
+
+    opener.on('click', function(e){
+        e.preventDefault();
+        body.addClass('mobile-overflow');
+        //popup.addClass('search-opened');
```
Comment (tolerants) [inline js/customscripts.js:84]: Have you temporary commented it or forgot to remove?

File: js/customscripts.js [hunk 5]
```diff
@@ -33,8 +33,69 @@ $( document ).ready(function() {
...
+        popup.fadeIn(300, function(){
+            input.focus();
+        });
+    });
+
+    close.on('click', function(e){
+        e.preventDefault();
+        //popup.removeClass('search-opened');
```
Comment (tolerants) [inline js/customscripts.js:92]: Have you temporary commented it or forgot to remove?

File: js/customscripts.js [hunk 6]
```diff
@@ -33,8 +33,69 @@ $( document ).ready(function() {
...
+        body.removeClass('mobile-overflow');
+
+        popup.fadeOut(300, function(){
```
Comment (tolerants) [inline js/customscripts.js:95]: Why do you need empty function here?

File: js/customscripts.js [hunk 7]
```diff
@@ -33,8 +33,69 @@ $( document ).ready(function() {
...
+        isScrolled = false,
+        searchContainer = $('.js-home-search'),
+        opener = $('.js-search-popup-opener'),
+        searchOffsetTop;
+
+    function handleScroll(){
+        pageOffsetTop = page.scrollTop();
+        searchOffsetTop = searchContainer.offset().top;
+
+        if(isScrolled && pageOffsetTop < searchOffsetTop) {
```
Comment (tolerants) [inline js/customscripts.js:58]:
```suggestion
        if (isScrolled && pageOffsetTop < searchOffsetTop) {
```

File: js/customscripts.js [hunk 8]
```diff
@@ -33,8 +33,69 @@ $( document ).ready(function() {
...
+    if (!homePage.length) return;
+
+    let page = jQuery(window),
+        pageOffsetTop,
+        isScrolled = false,
+        searchContainer = $('.js-home-search'),
+        opener = $('.js-search-popup-opener'),
+        searchOffsetTop;
+
+    function handleScroll(){
```
Comment (tolerants) [inline js/customscripts.js:54]:
```suggestion
    function handleScroll() {
```

**Review by tolerants — CHANGES_REQUESTED** [review]

File: css/components/_search-section.scss (hunk 2 above)
Comment (skliarzfort) [reply css/components/_search-section.scss:83]: it's a copy of search input with event listener. real input is hidden inside popup by default on mobile resolution

File: js/customscripts.js [hunk 9]
```diff
@@ -33,8 +33,66 @@ $( document ).ready(function() {
...
+    if (!homePage.length) return;
+
+    let page = jQuery(window),
+        pageOffsetTop,
+        isScrolled = false,
+        searchContainer = $('.js-home-search'),
+        opener = $('.js-search-popup-opener'),
+        searchOffsetTop;
+
+    function handleScroll (){
```
Comment (tolerants) [inline js/customscripts.js:54]:
```suggestion
    function handleScroll() {
```

File: css/components/_search-section.scss (hunk 2 above)
Comment (tolerants) [reply css/components/_search-section.scss:83]: Suggest to rename into `fake-input` for example.

**Review by tolerants — APPROVED** [review]

**Pull request merged**